NeoML/src/TraditionalML/ProblemWrappers.cpp

```diff
@@ -51,13 +51,13 @@ CMultivariateRegressionOverUnivariate::CMultivariateRegressionOverUnivariate(
 }
 
 // Gets the number of features
-int CMultivariateRegressionOverUnivariate::GetFeatureCount() const
+inline int CMultivariateRegressionOverUnivariate::GetFeatureCount() const
```

Comment:
Move to .inl or remove `inline`

Reply:
Why are you using a separate file for inline methods?

Reply:
For the structurization sake)